src/GeneralStateTestsFiller/stExample/09_revertUndoesAllFiller.yml

```diff
@@ -0,0 +1,93 @@
+# 04
```

Comment:
Can you update the comments at the top? I write the test number, then description, then what we should expect in storage.

Reply:
same for tests 11, 12, & 13!

Reply:
woops never read the comments at the top

Reply:
thought it was boilerplate